vulture_os/toolkit/api_parser/cisco_duo/cisco_duo.py

```diff
@@ -167,43 +167,30 @@ def get_logs(self, since=None, to=None, endpoint=None):
         query['sort'] = "ts:asc"
 
         return self.__execute_query(hostname, path, query)
-    
+
     def format_log(self, log):
         return json.dumps(log)
 
     def execute(self):
 
         for endpoint in self.ENDPOINTS:
```

Comment:
Warning here ! Updating the last_api_call in this for loop can be dangerous and won't work as expected if we add some other endpoints later.